Subject: [PATCH] weee
---
Index: src/test/java/com/example/ExamplePluginTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/com/example/ExamplePluginTest.java b/src/test/java/com/example/ExamplePluginTest.java
--- a/src/test/java/com/example/ExamplePluginTest.java	(revision a51715299fc8f857621e6d7e3547ad77d5e98aa3)
+++ b/src/test/java/com/example/ExamplePluginTest.java	(date 1688171863940)
@@ -1,5 +1,6 @@
 package com.example;
 
+import com.Gamepad.ExamplePlugin;
 import net.runelite.client.RuneLite;
 import net.runelite.client.externalplugins.ExternalPluginManager;
 
Index: runelite-plugin.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/runelite-plugin.properties b/runelite-plugin.properties
--- a/runelite-plugin.properties	(revision a51715299fc8f857621e6d7e3547ad77d5e98aa3)
+++ b/runelite-plugin.properties	(date 1688172233474)
@@ -1,5 +1,5 @@
-displayName=Example
-author=Nobody
-description=An example greeter plugin
+displayName=Gamepad
+author=graywes
+description=native gamepad support
 tags=
-plugins=com.example.ExamplePlugin
\ No newline at end of file
+plugins=com.example.ExamplePluginTest
\ No newline at end of file
Index: src/main/java/com/example/ExampleConfig.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/ExampleConfig.java b/src/main/java/com/Gamepad/ExampleConfig.java
rename from src/main/java/com/example/ExampleConfig.java
rename to src/main/java/com/Gamepad/ExampleConfig.java
--- a/src/main/java/com/example/ExampleConfig.java	(revision a51715299fc8f857621e6d7e3547ad77d5e98aa3)
+++ b/src/main/java/com/Gamepad/ExampleConfig.java	(date 1688172329872)
@@ -1,4 +1,4 @@
-package com.example;
+package com.Gamepad;
 
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
@@ -9,7 +9,7 @@
 {
 	@ConfigItem(
 		keyName = "greeting",
-		name = "Welcome Greeting",
+		name = "Welcome Greet",
 		description = "The message to show to the user when they login"
 	)
 	default String greeting()
Index: src/main/java/com/example/ExamplePlugin.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/ExamplePlugin.java b/src/main/java/com/Gamepad/ExamplePlugin.java
rename from src/main/java/com/example/ExamplePlugin.java
rename to src/main/java/com/Gamepad/ExamplePlugin.java
--- a/src/main/java/com/example/ExamplePlugin.java	(revision a51715299fc8f857621e6d7e3547ad77d5e98aa3)
+++ b/src/main/java/com/Gamepad/ExamplePlugin.java	(date 1688172437453)
@@ -1,4 +1,4 @@
-package com.example;
+package com.Gamepad;
 
 import com.google.inject.Provides;
 import javax.inject.Inject;
@@ -14,7 +14,7 @@
 
 @Slf4j
 @PluginDescriptor(
-	name = "Example"
+	name = "Examplie"
 )
 public class ExamplePlugin extends Plugin
 {
